ext/lib/gui/lvgl/lv_conf.h

```diff
@@ -0,0 +1,321 @@
+
+/*Text settings*/
+#define LV_TXT_UTF8           CONFIG_LVGL_UTF_8  /*Enable UTF-8 coded Unicode character usage */
+#define LV_TXT_BREAK_CHARS    " ,.;:-_"          /*Can break texts on these chars*/
+
+/*Graphics feature usage*/
+
+#define USE_LV_ANIMATION        CONFIG_LVGL_ANIMATION     /*1: Enable all animations*/
+#define USE_LV_SHADOW           CONFIG_LVGL_SHADOW        /*1: Enable shadows*/
+#define USE_LV_GROUP            CONFIG_LVGL_GROUP         /*1: Enable object groups (for keyboards)*/
+#define USE_LV_GPU              CONFIG_LVGL_GPU           /*1: Enable GPU interface*/
+#define USE_LV_REAL_DRAW        CONFIG_LVGL_DIRECT_DRAW   /*1: Enable function which draw directly to the frame buffer instead of VDB (required if LV_VDB_SIZE = 0)*/
+#define USE_LV_FILESYSTEM       CONFIG_LVGL_FILESYSTEM    /*1: Enable file system (required by images*/
+
+/*Compiler settings*/
+#define LV_ATTRIBUTE_TICK_INC                 /* Define a custom attribute to `lv_tick_inc` function */
+#define LV_ATTRIBUTE_TASK_HANDLER             /* Define a custom attribute to `lv_task_handler` function */
+#define LV_COMPILER_VLA_SUPPORTED    1        /* 1: Variable length array is supported*/
+
+/*================
+ *  THEME USAGE
+ *================*/
+#ifdef CONFIG_LGVL_THEMES
+
+#define USE_LV_THEME_TEMPL      0                           /*Just for test*/
+#define USE_LV_THEME_DEFAULT    CONFIG_LVGL_THEME_DEFAULT   /*Built mainly from the built-in styles. Consumes very few RAM*/
+#define USE_LV_THEME_ALIEN      CONFIG_LVGL_THEME_ALIEN     /*Dark futuristic theme*/
+#define USE_LV_THEME_NIGHT      CONFIG_LVGL_THEME_NIGHT     /*Dark elegant theme*/
+#define USE_LV_THEME_MONO       CONFIG_LVGL_THEME_MONO      /*Mono color theme for monochrome displays*/
+#define USE_LV_THEME_MATERIAL   CONFIG_LVGL_THEME_MATERIAL  /*Flat theme with bold colors and light shadows*/
+#define USE_LV_THEME_ZEN        CONFIG_LVGL_THEME_ZEN       /*Peaceful, mainly light theme */
+
+#else
+
+#define USE_LV_THEME_TEMPL      0       /*Just for test*/
+#define USE_LV_THEME_DEFAULT    0       /*Built mainly from the built-in styles. Consumes very few RAM*/
+#define USE_LV_THEME_ALIEN      0       /*Dark futuristic theme*/
+#define USE_LV_THEME_NIGHT      0       /*Dark elegant theme*/
+#define USE_LV_THEME_MONO       0       /*Mono color theme for monochrome displays*/
+#define USE_LV_THEME_MATERIAL   0       /*Flat theme with bold colors and light shadows*/
+#define USE_LV_THEME_ZEN        0       /*Peaceful, mainly light theme */
+
+#endif
+
+/*==================
+ *    FONT USAGE
+ *===================*/
+
+/* More info about fonts: https://littlevgl.com/basics#fonts
+ * To enable a built-in font use 1,2,4 or 8 values
+ * which will determine the bit-per-pixel */
+#define LV_FONT_DEFAULT        &lv_font_dejavu_20     /*Always set a default font from the built-in fonts*/
```

Comment:
Moved from #9149 
> @pizi-nordic
> You are always selecting lv_font_dejavu_20, but it can be compiled out basing on CONFIG_LVGL_BUILD_IN_FONT_20. Is that correct?

Reply:
made this configurable including a pointer to a custom font